lib/assessment.rb

```diff
@@ -14,13 +14,13 @@ def longest(enum)
   # Assume each entry implements a `+` method
   def sum(enum)
     # Your code goes here
-  end
-
+    enum.reduce { |memo, index| memo + index}
```

Comment:
Correct. This could have been written even more shortly if you used the `&` trick:
`enum.reduce(&:+)`